packages/pixeloven-cli/src/extensions/prettier.ts

```diff
@@ -0,0 +1,24 @@
+import { PixelOvenRunContext } from "../types";
```

Comment:
These three files are 90% the same. Any reason not to combine them to one and accept arguments for the differences?

Reply:
yeah I'll like do some cleanup once I figure out how to do tests and get parity with the old -tasks deal. Good call out!